shared/js/background/chrome-events.es6.js

```diff
@@ -217,6 +219,17 @@ chrome.webNavigation.onCommitted.addListener(details => {
             const whitelisted = settings.getSetting('whitelisted')
             const tabURL = new URL(details.url) || {}
             if (!whitelisted || !whitelisted[tabURL.hostname]) {
+                // Set variables, which are used in the fingerprint-protection script.
+                const variableScript = {
+                    'code': `
+                        try {
+                            var ua='${agentSpoofer.getAgent()}'
```

Comment:
🦹‍♂️ So… what's going to happen if I visit DDG 1000s of times using spoofed UA `' eval("hello world!");'`? Will it make it into here? Should we sanitize this?

Reply:
This is an awesome catch! An eval won't execute here, but a carefully crafted UA can get XSS on visited pages. So, if enough traffic was generated with a malicious UA (would need to be very significant), or a malicious userAgent file was served from our domain, this could be an issue. 

Unfortunately, UserAgents by RFC can contain any valid character, so it isn't as simple as a normal XSS filter.

What I did is strip out agent that contains single quotes in the UA, which I have not seen any valid user agents that include it, and is needed to break out of the assignment.

Reply:
Sounds good! Maybe we could also do some filtering on the back-end if it's simple enough?